rust-src/concordium_base/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Add a new module `cis3_types` that defines the interface types for CIS3
   compatible contracts.
+- Fix discrepancy in (de)serializing `Web3IdAttribute::Timestamp`s due to an unexpected breaking change introduced in an update to the `chrono` dependency.
```

Comment:
I think it would be good to point to the exact version of chrono with the breaking change, so we know we can't downgrade below this again.

Reply:
Good point